friends/friendModel.js

```diff
@@ -0,0 +1,41 @@
+const mongoose = require('mongoose');
+
+const FriendSchema = new mongoose.Schema({
+    firstName: {
+        type: String,
+        required: true
+    },
+    lastName: {
+        type: String,
+        required: true
+    },
+    age: {
+        type: Number,
+        required: true,
+        min: 1,
```

Comment:
Nice work! Theoretically the min/max shouldn't have any effect, since you validated that data in your post/put requests. However, this is still excellent practice and it's a good habit to think of your schemas this way.